src/main/java/com/google/devtools/build/lib/sandbox/AbstractSandboxSpawnRunner.java

```diff
@@ -125,16 +125,22 @@ private final SpawnResult run(
         sandbox.getSandboxExecRoot().getPathFile());
       String failureMessage;
       if (sandboxOptions.sandboxDebug) {
-        failureMessage =
-            CommandFailureUtils.describeCommandFailure(
-                true, sandbox.getArguments(), sandbox.getEnvironment(), execRoot.getPathString());
+      failureMessage =
+          CommandFailureUtils.describeCommandFailure(
+              true,
+              sandbox.getArguments(),
+              sandbox.getEnvironment(),
+              execRoot.getPathString(),
+              null);
```

Comment:
Why null? Why cannot we use the same value as in the branch below?

Reply:
In an ideal world, we'd have two different execution platforms here:
- The actual spawn's execution platform, which would describe the sandbox (and ideally give configuration for it)
- The sandbox's execution platform, which describes the environment that the sandbox container can run in.

We don't actually define either of these very well right now, but it's definitely wrong to assume they're the same, so it's best to just pass null and not describe it.

Reply:
(I see now that the two branches are subtly different in what they call `get*` on.)

But can the platforms between the sandbox and the action ever be different?

Reply:
One of my goals is to better couple execution strategies and platforms. Ideally, we'd end up with platform definitions like this (this is entirely a strawman, don't take any actual syntax seriously):

```
platform(name = "linux_sandbox",
  constraint_values = [...],
  execution_strategy = "linux-sandbox",
  execution_properties = """
    sandbox_base_dir: /tmp/sandbox
    sandbox_allow_networking: false
  """)

platform(name = "host",
  constraint_values = [...],
)
```

In this case, both of these are execution platforms, but "host" is the platform the sandbox runs on. They might have identical constraint_values, but they mean very different things.